cmd/main.go

```diff
@@ -143,6 +143,7 @@ var Options struct {
 	EnableElasticAPM               bool          `envconfig:"ENABLE_ELASTIC_APM" default:"false"`
 	WorkDir                        string        `envconfig:"WORK_DIR" default:"/data/"`
 	LivenessValidationTimeout      time.Duration `envconfig:"LIVENESS_VALIDATION_TIMEOUT" default:"5m"`
+	V1APIEnabled                   bool          `envconfig:"V1_API_ENABLED" default:"true"`
```

Comment:
Need to add it to https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml 
it will enable use to use it on staging and prod